cmd/yorkie/user/login.go

```diff
@@ -102,6 +107,5 @@ func init() {
 		false,
 		"Skip the TLS connection of the client",
 	)
-	cmd.MarkFlagsRequiredTogether("username", "password")
-	rootCmd.AddCommand(cmd)
```

Comment:
It seems that `username` should be a required field. Could we modify this using `cmd.MarkFlagsRequired("username")`? In the `StringVarP` description for `username`, perhaps we could change "(required if password is set)" to simply "required" to reflect this?